test/v1/controllers/RetrievePeriodSummaryControllerSpec.scala

```diff
@@ -38,22 +36,45 @@ import scala.concurrent.Future
 
 class RetrievePeriodSummaryControllerSpec
     extends ControllerBaseSpec
-    with MockEnrolmentsAuthService
-    with MockMtdIdLookupService
+    with ControllerTestRunner
     with MockRetrievePeriodSummaryService
     with MockRetrievePeriodSummaryRequestParser
-    with MockHateoasFactory
-    with MockAuditService
-    with MockIdGenerator {
+    with MockHateoasFactory {
 
-  private val nino          = "AA123456A"
-  private val businessId    = "XAIS12345678910"
-  private val periodId      = "2019-01-01_2020-01-01"
-  private val correlationId = "X-123"
-  private val taxYear       = "2023-24"
+  private val businessId = "XAIS12345678910"
+  private val taxYear    = "2023-24"
 
-  trait Test {
-    val hc: HeaderCarrier = HeaderCarrier()
+  trait Test extends ControllerTest {
```

Comment:
As per the comment on ListPeriodSummariesControllerSpec, please move the two traits after the tests and add type annotation for the vals.